layouts/_default/newsletters-signup.html

```diff
@@ -1,3 +1,10 @@
+{{ define "title" }}
+  <title>{{ .Param "title-tag" | default .LinkTitle }}</title>
+{{ end }}
+{{ define "body-class" }}
+  text-[16px] bg-beige font-serif leading-normal text-s-9
+{{ end }}
```

Comment:
I think if you set the default body-class in baseof, you won't need to add an override here.